src/main/resources/templates/400.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+
+<html xmlns:th="http://www.thymeleaf.org" xmlns="http://www.w3.org/1999/xhtml">
+<head th:include="main.html::head"></head>
+<body>
+<header th:replace="main.html::header"></header>
+
+<main id="main" role="main">
+    <div th:replace="main.html::phase"></div>
+    <h1>Bad Request</h1>
+    <p>Please check the request parameters.</p>
```

Comment:
Should we maybe be clearer about the exact error rather than having a generic response here?